js/languages/fr.js

```diff
@@ -0,0 +1,326 @@
+    wikipedia: 'Wikipedia',
+    'barcode-desc-after-link': '.',
+    'add-barcode': 'Ajouter un code-barres',
+    minscore: 'Calculateur de note de test final minimum',
+    'current-grade': 'La note actuelle: ',
+    'finals-worth': 'Partie de la note déterminée par le test final: ',
+    'min-grade': 'La note minimum acceptable: ',
+    map: 'Carte',
+    'for-image': "Pour l'image",
+    'image-instruct': ' Vous pouvez utiliser un doigt pour effectuer un panoramique ou deux pour faire pivoter, zoomer et effectuer un panoramique. Vous pouvez utiliser votre souris pour faire glisser, faire défiler pour zoomer ou cliquer avec le bouton droit pour faire pivoter.',
+    'for-gmaps': 'Pour Google Maps:',
+    'gmaps-instruct': " Fonctionne uniquement lorsque vous êtes connecté à Internet. Si vous êtes sur l'écran tactile et ne pouvez pas faire défiler vers le haut, essayez de faire défiler les boutons.",
+    'club-list': 'Liste des clubs',
+    'clubs-disclaimer-before-link': "Veuillez noter que les informations sur les clubs ont été extraites de la ",
+    'clubs-disclaimer-link': "liste de l'administration des clubs agréés en date du 13 Septembre, 2019",
+    'clubs-disclaimer-after-link': ', alors attribuez-leur des inexactitudes.',
+    'lists-disabled': "Vous avez désactivé les listes de club / personnel",
+    'turn-back-on': 'Réactiver',
+    'select-date': 'sélectionner une date',
+    events: 'Événements',
+    'staff-list': 'Liste du personnel',
+    'before-version': 'Vous utilisez le ',
+    'after-version': ' version.',
+    theme: 'Thème',
+    'light-theme': 'Thème clair',
+    'dark-theme': 'Thème foncé',
+    'neither-theme': 'Aucun thème',
+    'time-formatting': "Formatage de l'heure",
+    '12h': '12-heure',
+    '24h': '24-heure',
+    'title-compact': "Compacité du titre de l'onglet",
+    units: 'Unités incluses',
+    numbers: 'Chiffres uniquement',
+    'disable-lists': 'Désactiver les listes de club / personnel',
+    periods: 'Périodes',
+    'periods-desc': 'Cliquez sur le square coloré pour changer la couleur de la période.',
+    'control-centre': 'Centre de Contrôle',
+    'control-centre-desc': "Si l'UGWA ne se met pas à jour, essayez ces boutons.",
+    reload: 'Redémarrer UGWA',
+    'trick-cache': 'tromper le cache',
+    'staff-name': 'Nom du personnel',
+    close: 'Fermer',
+    'club-name': 'Nom du club',
+    ok: 'OK',
+    'add-ugwa': "Pour ajouter UGWA à votre écran d'accueil",
+    desktop: 'Ordinateur de Bureau',
+    'desktop-instruct': '—bookmark this page or pin this tab (right click on the tab and select "Pin tab")',
+    chromebook: 'Chromebook',
+    'chromebook-instruct': "—épingler cette page à l'étagère (menu > more tools > add to shelf)",
+    ios: 'iOS',
+    'ios-instruct': '—appuyez sur l\'icône de partage et sélectionnez "add to home screen"',

```

Comment:
This is just to escape the `'` to prevent it from ending the string (because the string is enclosed in single quotes); the backslash won't show in the app